Taorui huang

taoruihuang opened the pull request.

taoruihuang committed:
Refactored classes

taoruihuang committed:
Refactoring classes

taoruihuang committed:
Delete Object.py

taoruihuang committed:
Refactoring classes

taoruihuang committed:
Refactoring classes

taoruihuang committed:
Merge branch 'ped-grid-master' into taorui-huang

taoruihuang committed:
Refactoring classes

taoruihuang committed:
Updated Road.py & new Env

taoruihuang committed:
Sidewalks, Lane Position

taoruihuang committed:
Update Vehicle.py

taoruihuang committed:
Updated TwoLaneRoadEnv

taoruihuang committed:
inLane for Vehicles

taoruihuang committed:
Update TwoLaneRoadEnv.py

taoruihuang committed:
Update TwoLaneRoadEnv.py

adhocmaster commented on gym_minigrid/agents/Lane.py:
```diff
@@ -1,17 +1,24 @@
         bottomRight: Tuple[int, int],
         direction: int,
-        laneID: int
+        laneID: int,
```
add Road ID

adhocmaster commented on gym_minigrid/agents/Vehicle.py:
```diff
@@ -1,13 +1,24 @@
+        direction: int,
+        maxSpeed: float,
+        speed: float,
+        inLane: int
```
add road id

adhocmaster commented on gym_minigrid/envs/pedestrian/TwoLaneRoadEnv.py:
```diff
@@ -1,17 +1,90 @@
+        else:
+            logging.warn("Agent not in list")
+    
+    def forwardVehicle(self, agent: Vehicle):
```
forwardAgent should be enough

taoruihuang committed:
Refactored Lane and Vehicle

taoruihuang committed:
Revising method names

zarifikram committed:
Revert "Create test_benchmark.py"

This reverts commit a1bb7aa51c6f29d3b69b3e08caf41d07abb49886.

zarifikram committed:
Merge branch 'taorui-huang' into zarif2

zarifikram committed:
Bugs

taoruihuang committed:
Bugs

taoruihuang committed:
Update CrosswalkEnv.py

taoruihuang committed:
fixed some tests

taoruihuang committed:
Addressed self.vehicleAgents not attribute error

taoruihuang committed:
Update PedestrianEnv.py

zarifikram committed:
Merge branch 'taorui-huang' into zarif2

zarifikram committed:
Vehicle functioning

taoruihuang committed:
Added object coloring support

taoruihuang committed:
Updates to rendering

taoruihuang committed:
Fixed overlapping issue + added vehicle movement

taoruihuang committed:
Adjusted resetting

taoruihuang committed:
fixed 2 tests by copy-and-pasting from tests-fixes branch

taoruihuang committed:
Fixed test_2_agents test

taoruihuang committed:
fixed vertical movement

taoruihuang committed:
Create Crosswalk.py

taoruihuang committed:
Fixed movement

taoruihuang committed:
Experimented with coloring

taoruihuang committed:
Update test-TwoLaneRoadEnv.py

taoruihuang committed:
Added crosswalk class

taoruihuang committed:
crosswalk + sample environment

taoruihuang committed:
test environment

taoruihuang committed:
new test env + colors + building class

taoruihuang committed:
Update TwoLaneRoadEnv.py

zarifikram committed:
marged with thuang

zarifikram committed:
added stop go model vehicle and peds

Need to add vehicle following (when the trailing vehicle is faster). Need to add pedestrian calculating the vehicle distance

taoruihuang committed:
added stop go model vehicle and peds

Need to add vehicle following (when the trailing vehicle is faster). Need to add pedestrian calculating the vehicle distance

taoruihuang committed:
Create __init__.py

taoruihuang committed:
Added followFrontVehicleSpeed method

taoruihuang committed:
Notify when vehicle on sidewalk

taoruihuang committed:
Update PedGrid.py

taoruihuang committed:
vehicle on crosswalk alerts at each time step

taoruihuang committed:
Update PedGrid.py

taoruihuang committed:
Update Crosswalk.py

taoruihuang committed:
Added properties to crosswalk

taoruihuang committed:
Quick bug fix + test follow front vehicle speed

zarifikram committed:
Merge branch 'taorui-huang' into zarif2

zarifikram committed:
Create ._stopGoVehicle.py

taoruihuang committed:
Update IDs

taoruihuang committed:
Update TwoLaneRoadEnv.py

taoruihuang committed:
fixed wording

taoruihuang committed:
Added lastVehiclesCrossed to Crosswalk class

taoruihuang committed:
introduced unfinished updateIncomingVehicles

zarifikram committed:
Merge branch 'taorui-huang' into zarif2

zarifikram committed:
Bug on test-StopGoModel.py

zarifikram committed:
Skeleton added for pedstop

Terry :
1. We need to find out the vehicle in the crosswalk given the lane of ped

2. We need to find the closest incoming vehicle in the lane of ped. Lane ID affects which side of crosswalk we need to look at

taoruihuang committed:
Squashed commit of the following:

taoruihuang committed:
Merge branch 'zarif2' into taorui-huang

taoruihuang committed:
finished updateIncomingVehicles

taoruihuang committed:
fixed updateIncomingVehicles

taoruihuang committed:
Update Crosswalk.py

taoruihuang committed:
Alternative solution to finding distance betw. 2 vehicles

taoruihuang committed:
bug fixes

taoruihuang committed:
bug fixes

zarifikram committed:
Almost working

zarifikram committed:
Update test-StopGoModel.py

taoruihuang committed:
Fixed vehicles not moving right away

taoruihuang committed:
Update Agent.py

taoruihuang committed:
Change back to original minigrid library code

taoruihuang committed:
revamped metric collection

taoruihuang committed:
Update MetricCollector.py

taoruihuang committed:
added TTC env + speed up rendering

taoruihuang committed:
new SimplePedAgent

taoruihuang committed:
new TTC model

taoruihuang committed:
working TTC model

taoruihuang committed:
working TTC Model

taoruihuang committed:
TTC update

taoruihuang committed:
better TTC model

taoruihuang committed:
adjusted TTC model

taoruihuang committed:
Update test-TTCModel.py

taoruihuang committed:
Update TTC Model.ipynb

taoruihuang committed:
Update test-interspersed.py
